gnomad_qc/v5/sample_qc/hard_filters.py

```diff
@@ -59,11 +60,21 @@ def compute_aou_sample_qc(
         vds, intervals=telomeres_and_centromeres.ht(), keep=False
     )
 
+    logger.info("Excluding loci with more than 100 alternative alleles...")
+    vmt = vds.variant_data
+    vmt = vmt.annotate_rows(n_unsplit_alleles=hl.len(vmt.alleles))
+    vmt = vmt.filter_rows(vmt.n_unsplit_alleles < 101)
```

Comment:
```suggestion
    vmt = vmt.filter_rows(vmt.n_unsplit_alleles < 100)
```
I don't think we need to keep n_unsplit_alleles ==  100 (though using 101 or 100 here shouldn't make a huge difference)

Reply:
Let's align with AoU? Because they used alt alleles >100 and I got the same count.